src/shogun/neuralnets/NeuralNetwork.h

```diff
@@ -442,6 +442,20 @@ friend class CDeepBeliefNetwork;
 	 * default value is -1
 	 */
 	float64_t gd_error_damping_coeff;
+
+	//@{ 
```

Comment:
Grouping similar members together. Both of those are only used for RmsProp.  https://www.stack.nl/%7Edimitri/doxygen/manual/grouping.html#memgroup